examples/run.rs

```diff
@@ -0,0 +1,87 @@
+use async_trait::async_trait;
+use faktory::{Client, Job, JobRunner, Worker};
+use std::io::{Error as IOError, Result as IOResult};
+use std::sync::Arc;
+use tokio::sync::mpsc;
+use tokio::time;
+
+pub struct JobHandler<T> {
+    chan: Arc<mpsc::Sender<T>>,
+}
+
+impl<T> JobHandler<T> {
+    pub fn new(chan: &Arc<mpsc::Sender<T>>) -> Self {
+        Self {
+            chan: Arc::clone(chan),
+        }
+    }
```

Comment:
```suggestion
    pub fn new(chan: Arc<mpsc::Sender<T>>) -> Self {
        Self { chan }
    }
```
nit: the convention in Rust is to allow the caller to be in charge of getting an owned instance of a value

Reply:
Indeed, just like with the `&str` vs `String` case we came across elsewhere.